pkg/config/data.go

```diff
@@ -181,13 +191,13 @@ type File struct {
 	ConfigVersion int                 `toml:"config_version"`
 	Fastly        Fastly              `toml:"fastly"`
 	Language      Language            `toml:"language"`
+	Profiles      Profiles            `toml:"profile"`
 	StarterKits   StarterKitLanguages `toml:"starter-kits"`
-	User          User                `toml:"user"`
 	Viceroy       Viceroy             `toml:"viceroy"`
 
 	// We store off a possible legacy configuration so that we can later extract
 	// the relevant email and token values that may pre-exist.
-	Legacy LegacyFile `toml:"legacy"`
```

Comment:
The `user` field in the config.toml is now considered 'legacy' and will be auto-migrated to the new `[profiles]` section. 

A long time ago the `user` field was considered _new_ and the _very original_ config.toml format had a user's email and token in the top level file. So when we first switched to `user` we introduced a `legacy` field into the toml. 

That was a long time ago and we don't need to support `legacy` any more.